filesystem/filesystem.go

```diff
@@ -192,20 +197,36 @@ func mountRootfs(ipList []string, target string, cluster *v1.Cluster) error {
 	return nil
 }
 
-func CopyFiles(ssh ssh.Interface, isRegistry bool, ip, src, target string) error {
+func CopyFiles(ssh ssh.Interface, isRegistry bool, addrs *[]*[]net.Addr, ip, src, target string) error {
```

Comment:
*[]*[]net.Addr I think this argument is really not very friendly. Maybe CopyFile funcion accepts `isLocalAddr bool` is better? And using IsLocalAddrs instead GetLocalHostAddres, because mount really not care about what the local ip List is.